packages/ng/styles/definitions/option/_option-item.scss

```diff
@@ -2,22 +2,21 @@
 @use '@lucca-front/icons/src/commons/utils/icon';
 @use '@lucca-front/scss/src/commons/utils/a11y';
 
-@include namespace.appendRootVars {
```

Comment:
It seems to have side effect as some variables are not defined anymore. e.g. in old Establishement: 
<img width="839" height="568" alt="image" src="https://github.com/user-attachments/assets/bfcd32b3-a418-4de4-8bb4-8d9bdac0adb4" />

Group name used to have padding:
<img width="293" height="437" alt="image" src="https://github.com/user-attachments/assets/ec367366-8b53-4cfc-9da2-82820f00a9bd" />